dev/fattest.simplicity/build.gradle

```diff
@@ -44,12 +44,14 @@ jar {
 
 configurations {
     arquillianFeature
-    arquillianJakartaFeature
+    arquillianJakartaFeature20
+    arquillianJakartaFeature21
```

Comment:
Do we really need to continue supporting version 2.0 as well as 2.1?

Reply:
Would have to update all TCK projects to use 2.1 in order to no longer use 2.0.  The feature version number changed to 2.1 so would need to update all TCK server.xml files to 2.1.  Could do that in this PR or could do it in another PR.

Reply:
Created https://github.com/OpenLiberty/open-liberty/pull/22176 to do that work and removed the build.gradle change from this PR.